.gitignore

```diff
@@ -120,3 +120,6 @@ Thumbs.db
 
 .nx/cache
 .nx/workspace-data
+
+.cursor/rules/nx-rules.mdc
+.github/instructions/nx.instructions.md
```

Comment:
Seems like the above file (`.github/instructions/nx.instructions.md`) is ignored, should we just delete it? If so, you can delete the file directly from GitHub UI 🙂

Reply:
Done